### PR TITLE
Update f5-ctlr-agent to include whitelist update cleanup

### PR DESCRIPTION
**Pull request by russokj**

### CODE REVIEW

**Comment by russokj** [comment]
I need someone to approve and submit this.  Thanks

**Comment by ryan-talley** [comment]
agent-runtime-requirements.txt in the f5-ctlr-agent that this points to references git+https://github.com/russokj/f5-cccl.git@7109ac713af422af18206bcf5db3b500a2155a4a#egg=f5-cccl fix this to point to f5devcentral CCCL

**Comment by russokj** [comment]
Wow. Good catch.  Definitely a cut and paste error

**Comment by ryan-talley** [comment]
Ya I have definitely done that before too.

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
--e git+https://github.com/f5devcentral/f5-ctlr-agent.git@d6af54918f7523479f84f0fedee995452a7aed7e#egg=f5-ctlr-agent
+-e git+https://github.com/f5devcentral/f5-ctlr-agent.git@f6084d029514cd47252c6b1cc457e086b5d46d78#egg=f5-ctlr-agent
```
Comment (ryan-talley) [inline requirements.txt]: shouldn't this be at commit ffe7c22288502ca45f14bcf2b6d93295ca58a593 thats what top of master is at right now for the f5-ctlr-agent?

Comment (russokj) [reply requirements.txt]: I have f5-ctlr-agent TIP as ffe7c22288502ca45f14bcf2b6d93295ca58a593
and CCCL TIP as  5fce7b2d70679bf5b775e487df7b05ede4e0303b

**Comment by russokj** [comment]
I don't know what I'm doing.  See if it's right now.

**Comment by ryan-talley** [comment]
Good now I approved

**Comment by russokj** [comment]
Okay.  I'm done to 2%.  Can you merge when the check completes.

**Comment by ryan-talley** [comment]
Can do

**Comment by ryan-talley** [comment]
It will be later tonight if that's ok I have to leave now and won't be online until later.

**Pull request merged**